main.py

```diff
@@ -37,10 +39,12 @@ def reEmail(fname):
 def reAward(fname):
   fhand = open(fname, 'r')
   for line in fhand:
-    extr = re.findall('.*', line)
+    extr = re.findall('-.*', line)
```

Comment:
Your regex will extract EXTRA space in front of the award winner.  "  Usain Bolt"
Use the brackets ( ).

extr = re.findall('- (.*)', line)